### DISCUSSION TITLE
storageManager: save ids

### DISCUSSION THREAD

**Discussion by dcr31000** [GrapesJS/grapesjs#3748] [Q&A]
Hello,

I'm using the remote storageManager option to load and save templates. But this don't save the unique id (set by default or by me) on dom components.

`storageManager: {
				type: 'remote',
				autoload: true,
				stepsBeforeSave: 1,
				contentTypeJson: true,
				urlStore: '...,
				urlLoad: '...}',
				storeComponents: false,
				storeStyles: false,
				storeHtml: true,
				storeCss: true,
			},`

I want to save unique id for future processing.

Thanks

**Comment by artf** [comment]
IDs are stored in `attributes`

**Reply by dcr31000** [reply]
Is it possible to store an attribute in html ? I need it to parse the dom in future processing and detect if a node is in another one. Thanks

**Reply by dcr31000** [reply]
@artf Is it possible to retrieve the html with all attributes to parse dom with another tool ? thanks
